R/pca_sparse_bayes.R

```diff
@@ -0,0 +1,243 @@
+#' Sparse, Bayesian PCA Signal Extraction
```

Comment:
```suggestion
#' Sparse Bayesian PCA Signal Extraction
```
looks a little weird to me? :thinking: